FeedStoreChallenge/ManagedCache.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  ManagedCache+CoreDataClass.swift
+//  FeedStoreChallenge
+//
+//  Created by Andreas Link on 13.06.21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+//
+
+import Foundation
+import CoreData
+
+@objc(ManagedCache)
+internal class ManagedCache: NSManagedObject {
+	@NSManaged internal var timestamp: Date
+	@NSManaged internal var feed: NSOrderedSet
+
+	static func fetch(in context: NSManagedObjectContext) throws -> ManagedCache? {
+		let request = NSFetchRequest<ManagedCache>(entityName: String(describing: ManagedCache.self))
+		return try context.fetch(request).first
```

Comment:
To slightly increase CoreData performance `request.returnsObjectsAsFaults` can be set to `false`.
```suggestion
		let request = NSFetchRequest<ManagedCache>(entityName: String(describing: ManagedCache.self))
		request.returnsObjectsAsFaults = false
		return try context.fetch(request).first
```
According to the [documentation](https://developer.apple.com/documentation/coredata/nsfetchrequest/1506756-returnsobjectsasfaults)
> When you execute a fetch, by default `returnsObjectsAsFaults` is `true`; Core Data fetches the object data for the matching records, fills the row cache with the information, and returns managed object as faults. These faults are managed objects, but all of their property data resides in the row cache until the fault is fired. When the fault is fired, Core Data retrieves the data from the row cache. ... If you need to access the property values from the returned objects (for example, if you iterate over all the objects to calculate the average value of a particular attribute), then it is more efficient to set `returnsObjectsAsFaults` to `false` to avoid the additional overhead.